### ISSUE TITLE
ethcontract-generate fails to compile 

### ISSUE THREAD

**Issue by dvec** [cowprotocol/ethcontract-rs#671]
```
error: 1 positional argument in format string, but no arguments were given
   --> /Users/username/.cargo/registry/src/github.com-1ecc6299db9ec823/ethcontract-generate-0.15.3/src/source.rs:103:46
    |
103 |             .map_err(|_| anyhow!("root path '{}' is not absolute"))?;
    |                                              ^^

error: could not compile `ethcontract-generate` due to previous error
```

I was able to fix this by changing `anyhow` crate version in `Cargo.lock` from `1.0.48` to `1.0.45`. 
This issue was fixed by @dtolnay in be8b6f69a0f17963e6e823b551467d4709bee7fd so I believe crates.io version should be updated.

**Comment by bh2smith** [comment]
Also reported here

https://github.com/gnosis/gp-v2-services/pull/1427#issuecomment-975909557

**Comment by vkgnosis** [comment]
You are right that we need to make a new release.

**Comment by andschneider** [comment]
I ran into this issue and adding the `--locked` flag to my build fixed it. e.g. `cargo install --locked`

**Comment by nlordell** [comment]
This should be fixed in `ethcontract v0.15.4`.
